frontend/src/Component/Share/Navbar/Navbar.jsx

```
@@ -0,0 +1,97 @@
+            }
+        };
+        document.addEventListener('mousedown', handleClickOutside);
+        return () => document.removeEventListener('mousedown', handleClickOutside);
+    }, []);
+
+    return (
+        <nav className="bg-indigo-800 text-white p-4 shadow-lg sticky top-0 z-50">
+            <div className="container mx-auto flex justify-between items-center">
+                
+                {/* Left Side - Mobile Menu Button */}
+                <button className="text-white md:hidden" onClick={() => setMenuOpen(!menuOpen)}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)